app/src/main/kotlin/com/wire/android/ui/calling/CallActivity.kt

```diff
@@ -85,6 +89,7 @@ class CallActivity : AppCompatActivity() {
 
         setUpCallingFlags()
         setUpScreenShootPreventionFlag()
+        lifecycle.addObserver(currentScreenManager)
```

Comment:
question: won't that create a race condition between `CallActivity` and `WireActivity` if both Activities lifecycles are observed at the same time by the `currentScreenManager`? Shouldn't we also `removeObserver` when opening and closing another Activity?

Reply:
According to the documentation:
![image](https://github.com/wireapp/wire-android/assets/30429749/7a256aa9-a4b4-40cd-8926-846b3c367976)
so first `onStart` from new Activity and then `onStop` from old one. 
We have `visibilityCount` there so it shouldn't have a negative impact in case of switching between activities, but still if we have multiple Activities now we should also remove these observers when Activity is destroyed. 
Also, logs from `CurrentScreenManager` will probably become hard to understand as they don't say which Activity is stopped or started, so when opening second Activity, in logs we will get `onStart` and then `onStop` so from logs it will look like app is not visible, but it's only the previous activity that has become stopped, not the whole app. 🤔